cmd/podman/start.go

```diff
@@ -141,7 +143,11 @@ func startCmd(c *cli.Context) error {
 		if lastError != nil {
 			fmt.Fprintln(os.Stderr, lastError)
 		}
-		lastError = ctr.Cleanup()
+		// We can only do this if we attached
+		// Otherwise the container is probably still running
+		if attach && tty {
+			lastError = ctr.Cleanup()
```

Comment:
You need this code correct?
			if lastError != nil {
				fmt.Fprintln(os.Stderr, lastError)
			}

Reply:
I think we don't need it because multiple container conflicts with --attach, so if we're in this block we should not have been in the loop multiple times

Reply:
Add comment then so future rhatdan does not add this.